webviz_config/_theme_class.py

```diff
@@ -1,3 +1,6 @@
+from copy import deepcopy
```

Comment:
Since we only use `deepcopy` once, consider using `import copy` here and then typing `copy.deepcopy()` further below where it is used.